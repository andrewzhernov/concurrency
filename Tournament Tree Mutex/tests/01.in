2 500000 0
