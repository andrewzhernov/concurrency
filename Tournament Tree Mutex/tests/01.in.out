Testing with parameters: n_threads=2, n_iters=500000, n_shareds=0
Starting 2 threads ... 
Done
Joining 2 threads ... 
Done
Checking test invariant ... 
Done
OK
Testing with parameters: n_threads=2, n_iters=500000, n_shareds=0
Starting 2 threads ... 
Done
Joining 2 threads ... 
Done
Checking test invariant ... 
Done
OK
Testing with parameters: n_threads=2, n_iters=500000, n_shareds=0
Starting 2 threads ... 
Done
Joining 2 threads ... 
Done
Checking test invariant ... 
Done
OK
